tutor/templates/apps/openedx/settings/cms/development.py

```diff
@@ -5,6 +5,9 @@
 LMS_BASE = "{{ LMS_HOST }}:8000"
 LMS_ROOT_URL = "http://" + LMS_BASE
 
+CMS_BASE = "{{ CMS_HOST}}:8001"
+CMS_ROOT_URL = "http://{}".format(CMS_BASE)
```

Comment:
Nit: let's make this match the style of LMS_BASE above (functionally equivalent)
```suggestion
CMS_BASE = "{{ CMS_HOST }}:8001"
CMS_ROOT_URL = "http://" + CMS_BASE
```

Reply:
Oh, sure. I literally just copy-pasted these lines from the other config file.